arrows/mvg/camera_options.cxx

```diff
@@ -0,0 +1,212 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief Implementation of MVG camera options.
+ */
+
+#include "camera_options.h"
+
+#define MVG_ENUM_HELPERS(NS, ceres_type)                                \
+namespace kwiver {                                                      \
+namespace vital {                                                       \
+                                                                        \
+template<>                                                              \
+config_block_value_t                                                    \
+config_block_set_value_cast(NS::ceres_type const& value)                \
+{                                                                       \
+  return NS::ceres_type##ToString(value);                               \
+}                                                                       \
+                                                                        \
+template<>                                                              \
+NS::ceres_type                                                          \
+config_block_get_value_cast(config_block_value_t const& value)          \
+{                                                                       \
+  NS::ceres_type cet;                                                   \
+  if(!NS::StringTo##ceres_type(value, &cet))                            \
+  {                                                                     \
+    VITAL_THROW( bad_config_block_cast,value);                          \
+  }                                                                     \
+  return cet;                                                           \
+}                                                                       \
+                                                                        \
+}                                                                       \
+                                                                        \
+namespace arrows {                                                      \
+namespace mvg {                                                       \
```

Comment:
missing a few whitespaces to align the '\\'

Reply:
corrected spacing to align \